website/docs/users-sources/sources/social-logins/lark/index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Lark(FeiShu)
+support_level: community
+---
+
+Allows users to authenticate using their Lark(Feishu) credentials.
+
+## Preparation
+
+The following placeholders are used in this guide:
+
+- `authentik.company` is the FQDN of the authentik installation.
+
+:::caution
+Administrator role for Lark(Feishu) is required to create clients on Lark(Feishu) developer platform.
+:::
+
+## Lark(Feishu)
+
+1. Login into ![Lark Open Platform](https://open.feishu.cn/).
+
+2. Click **Create Custom App** to create a new application.
+
+![Lark App Creation](./lark1.png)
+
+3. After creation, in the **Credentials & Basic Info**, find the **APP ID** and **APP Secret** and _save it for later_
+   ![Get APP Id and APP Secret](./lark2.png)
+
+4. Go to security settings.Insert the redirect URL: `https://authentik.company/source/oauth/callback/lark/`
+   ![Add Redirect URL](./lark3.png)
+
+5. Publish your application, changes only take effect after you have published.
+
+![Publish Appclication](./lark4.png)
+
+## authentik
+
+6. Under _Directory -> Federation & Social login_ Click **Create > Lark OAuth Source**
+
+![Lark OAuth Source](./lark5.png)
```

Comment:
```suggestion
```

Reply:
You have the same image, populated with the inputs, further down the page. Let's just have it there; we do not need both.